spot-ml/src/main/scala/org/apache/spot/SuspiciousConnectsArgumentParser.scala

```diff
@@ -95,6 +96,10 @@ object SuspiciousConnectsArgumentParser {
       action((x, c) => c.copy(hdfsScoredConnect = x)).
       text("HDFS path for results")
 
+    opt[String]("tempmodel").required().valueName("<hdfs path>").
```

Comment:
going forward... we might want to think of a way to reduce the CLIs for this jar.... we are nearing the "22 field limit" that Scala has for its build in tuples and case classes